client/app/src/shared/partials/tip-comments/tip-comments.component.html

```diff
@@ -34,12 +33,13 @@
                         </span>
                         <br/>
                     </div>
-                    <div *ngIf="comment.author_id" class="message">
+                    <div class="message">
```

Comment:
Since `comment.author_id` is referenced within this div, is this change not going to break anything?